scripts/provision-rabbitmq.sh

```diff
@@ -37,6 +37,14 @@ provision_rabbitmq_ubuntu14 () {
     # install RabbitMQ server:
     sudo DEBIAN_FRONTEND=noninteractive apt-get -y install \
          rabbitmq-server
+
+    # allow remote connections for guest user
+    sudo sudo chmod 777 -R /etc/rabbitmq
```

Comment:
I am not sure if this is a good idea.

I would go with something like:
```bash
echo '[{rabbit, [{loopback_users, []}]}].' | sudo tee -a /etc/rabbitmq/rabbitmq.config
```

Reply:
@jirikuncar except that one (good catch BTW), do you see any direct problems with this PR that would prevent you from merging it? 

as @diegodelemos wrote in his comment, https://github.com/inveniosoftware/invenio/issues/3691#issuecomment-355896393 , from what I understand, he considers this PR/branch to be abandoned. From my perspective, it just needs a few simple fixes and it could be committed, so Ubuntu 16.04 users could benefit from easy installation of Invenio. 

@jirikuncar , what are your thoughts on that?

Reply:
I don't see any other blocker.